src/Button.js

```diff
@@ -112,11 +115,16 @@ Button.propTypes = {
    */
   onClick: PropTypes.func,
   tooltip: PropTypes.string,
-  waves: PropTypes.oneOf(constants.WAVES)
+  waves: PropTypes.oneOf(constants.WAVES),
+  /**
+   * Click-Only
+   */
+  clickOnly: PropTypes.bool
 };
 
 Button.defaultProps = {
-  node: 'button'
+  node: 'button',
+  clickOnly: false
```

Comment:
I'm not sure we need a default prop for this guy, if he's not supplied the effect will be the same.